### PR TITLE
Rebase polaris-service-quarkus

### PR DESCRIPTION
**Pull request by jbonofre**
This PR rebases `polaris-service-quarkus` based on `polaris-service` (coming from `main` branch).

### CODE REVIEW

**Review by dimas-b — APPROVED** [review]
LGTM :+1:

**Comment by jbonofre** [comment]
@dimas-b I have 5 tests failing on `PolarisServiceImplIntegrationTest`.

File: polaris-service-quarkus/build.gradle.kts [hunk 1]
```diff
@@ -220,6 +220,15 @@ tasks.withType(Test::class.java).configureEach {
   addSparkJvmOptions()
 }
 
+tasks.named<Test>("test").configure {
+  if (System.getenv("AWS_REGION") == null) {
+    environment("AWS_REGION", "us-west-2")
+  }
+  jvmArgs("--add-exports", "java.base/sun.nio.ch=ALL-UNNAMED")
+  useJUnitPlatform()
+  maxParallelForks = 4
```
Comment (adutra) [inline polaris-service-quarkus/build.gradle.kts:229]: We must be careful with many forks running Quarkus tests, you could experience OOM issues. But let's leave this for now if tests are passing.

**Review by adutra — APPROVED** [review]

File: polaris-service-quarkus/build.gradle.kts (hunk 1 above)
Comment (jbonofre) [reply polaris-service-quarkus/build.gradle.kts:229]: Good point. I can decrease this value to 2 (or even 1) for now.

**Pull request merged**